opm-core/Installation.rst

```diff
@@ -131,12 +122,17 @@ Then, in your Nagios main configuration file, make sure the following parameter
             service_format_template = "DATATYPE::SERVICEPERFDATA\tTIMET::$icinga.timet$\tHOSTNAME::$host.name$\tSERVICEDESC::$service.name$\tSERVICEPERFDATA::$service.perfdata$\tSERVICECHECKCOMMAND::$service.check_command$\tHOSTSTATE::$host.state$\tHOSTSTATETYPE::$host.state_type$\tSERVICESTATE::$service.state$\tSERVICESTATETYPE::$service.state_type$\tSERVICEOUTPUT::$service.output$"
         }
 
-    Icinga2 has different macros names than Nagios, for complete list see
+    Icinga2 has different macros names than Nagios. For a complete list see
     `documentation <http://docs.icinga.org/icinga2/latest/doc/module/icinga2/toc#!/icinga2/latest/doc/module/icinga2/chapter/monitoring-basics#host-runtime-macros>`_.
 
+
+.. note::
+
+    Beware: the perfdata files accumulate very quickly if not consumed by the ``nagios_dispatcher`` script.
```

Comment:
I'd say that they **can** accumulate very quickly.

Reply:
done